logfile_schema.py

```diff
@@ -27,10 +27,38 @@
 
```

Comment:
Can't comment line 76 for whatever reason ever:
`errors_unique` variable is not used.